modules/terrain/src/lib/parse-terrain.js

```diff
@@ -61,23 +61,43 @@ function getMeshAttributes(vertices, terrain, tileSize, bounds) {
   };
 }
 
-function getMartiniTileMesh(terrainImage, terrainOptions) {
+/**
+ * Returns generated mesh object from image data
+ *
+ * @param {Array} terrainImage terrain image data
```

Comment:
I think this type is wrong? You declare `terrainImage` as an `Array`, but then you do

```js
  const {data, width, height} = terrainImage;
```